bin/lib/fetchContent.js

```diff
@@ -0,0 +1,91 @@
+const fetch = require('node-fetch');
+const debug = require('debug')('bin:lib:fetchContent');
+
+const     extractUuid = require('./extract-uuid');
+const    parseRSSFeed = require('./parse-rss-feed');
+const individualUUIDs = require('./individualUUIDs');
+
+const CAPI_KEY = process.env.CAPI_KEY;
+if (! CAPI_KEY ) {
+	throw new Error('ERROR: CAPI_KEY not specified in env');
+}
+
+const CAPI_PATH = 'http://api.ft.com/enrichedcontent/';
```

Comment:
that's only true if the url format doesn't change. I would have thought that something as big as an API change would be worth doing the equivalent of a local test and then deploy of a code change.